module/documents/item.mjs

```diff
@@ -1571,9 +1571,9 @@ export default class Item5e extends SystemDocumentMixin(Item) {
         else level = this.actor.system.details.spellLevel;
         rollConfigs.forEach(c => this._scaleCantripDamage(c.parts, scaling.formula, level, rollData));
       }
-      else if ( spellLevel && (scaling.mode === "level") && scaling.formula ) {
+      else if ( spellLevel && (scaling.mode === "level") ) {
```

Comment:
Does it break anything if we attempt scaling `undefined` or blank? Otherwise we can just guard it.
```suggestion
      else if ( spellLevel && (scaling.mode === "level") && (scaling.formula || c.parts.length) ) {
```